app/styles/themes/_dark.scss

```diff
@@ -16,8 +16,8 @@ body.theme-dark {
   --text-secondary-color: $gray-400;
   --background-color: $gray-900;
 
-  --button-background: black;
-  --button-hover-background: darken($gray-900, 5%);
+  --button-background: $blue;
+  --button-hover-background: lighten($blue, 5%);
```

Comment:
How does changing this to `darken` look? I feel like it might look better that way because it would be more symmetrical since the button color is always getting closer to the background color on hover.

Reply:
I really like your explanation of getting closer to the background color! But, when I tried this out it felt a little off. I think it's because the button is so familiar from the light theme that it's weird to see it with a different behavior.